challenge.rb

```diff
@@ -67,6 +93,13 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
-# Replace this comment with your code.
+def fizz_buzz min, max
+  i = min
+  while i <= max do
+    fizz_buzz_calculator(i)
```

Comment:
And this is why it matters. If you don't return anything from your method, you can't plug that method's output into any other methods.